src/API/Site/Controllers/Ads/RecommendationsController.php

```diff
@@ -0,0 +1,203 @@
+					'permission_callback' => $this->get_permission_callback(),
+					'args'                => $this->get_collection_params(),
+				],
+				'schema' => $this->get_api_response_schema_callback(),
+			]
+		);
+	}
+
+	/**
+	 * Get the query params for collections.
+	 *
+	 * @return array
+	 */
+	public function get_collection_params(): array {
+		return [
+			'type' => [
+				'type'        => 'string',
+				'description' => __( 'Filter recommendations by type', 'google-listings-and-ads' ),
+				'required'    => false,
+			],
+			'id'   => [
+				'type'        => 'integer',
+				'description' => __( 'Filter recommendations by unique id', 'google-listings-and-ads' ),
+				'required'    => false,
+			],
+		];
+	}
+
+	/**
+	 * Get the callback function for the list accounts request.
+	 *
+	 * @return callable
+	 */
+	protected function get_recommendations_callback(): callable {
+		return function ( Request $request ) {
+			try {
+				// Checks if the ads account is connected; exits early if not connected to prevent further execution.
+				$account_status = $this->account->get_connected_account();
+				if ( isset( $account_status['status'] ) && 'connected' !== $account_status['status'] ) {
+					return new Response(
+						[ 'message' => __( 'No connected Ads account found.', 'google-listings-and-ads' ) ],
+						403
+					);
+				}
+
+				$type = $request->get_param( 'type' );
+
+				// Only support filtering by type IMPROVE_PERFORMANCE_MAX_AD_STRENGTH for now.
+				if ( $type && $type !== 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH' ) {
+					return new Response( [] );
+				}
+
+				// TODO: Replace this with real data from AdsRecommendations service.
+				$recommendations = [
+					[
+						'id'              => 1,
+						'type'            => 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH',
+						'resource_name'   => 'customers/123/recommendations/1',
+						'campaign_id'     => 100,
+						'campaign_name'   => 'Test Campaign',
+						'campaign_status' => 'ENABLED',
+						'last_synced'     => gmdate( 'c' ),
+					],
+					[
+						'id'              => 2,
+						'type'            => 'IMPROVE_PERFORMANCE_MAX_AD_STRENGTH',
+						'resource_name'   => 'customers/123/recommendations/2',
+						'campaign_id'     => 101,
+						'campaign_name'   => 'Another Campaign',
+						'campaign_status' => 'PAUSED',
+						'last_synced'     => gmdate( 'c' ),
+					],
+				];
+
+				// If type is set, filter by type (only IMPROVE_PERFORMANCE_MAX_AD_STRENGTH is supported).
+				if ( $type ) {
+					$recommendations = array_filter(
+						$recommendations,
+						static function ( $rec ) use ( $type ) {
+							return $rec['type'] === $type;
+						}
+					);
+				}
+
+				// Filter by id if provided.
+				$id = $request->get_param( 'id' );
+				if ( $id ) {
+					$recommendations = array_filter(
+						$recommendations,
+						static function ( $rec ) use ( $id ) {
+							return (int) $rec['id'] === (int) $id;
+						}
+					);
+				}
```

Comment:
Ideally, filtering by type will be handled by the `AdsRecommendations::get_recommendations` method. We might want to enhance that method to also accept an ID argument so it can handle both.